www/less/_app.variables.less

```diff
@@ -122,6 +122,7 @@
 @login-input-font-color: @color-bali-hai-blue-s25;
 @login-font-color: @light-text;
 @login-input-border-custom-color: @light-text;
+@login-input-border-custom-mobile-color: #507299;
```

Comment:
Could you please create base variable for this color?
According to www.color-blindness.com